### ISSUE TITLE
moviepy integration

### ISSUE THREAD

**Issue by stonebig** [hhatto/pgmagick#27]
to propose pgmagick as an option to moviepy, it seems the following options would be nice to have:
- interline-spacing
- kerning

I didn't see them in current github.  If they are not there, would it be possible/easy to add them ?

**Comment by stonebig** [comment]
Apparently, these options were added to ImageMagick API only with version .

````
Re: interline spacing, interword spacing, kerning
Quote
Post  by dlemstra » 2014-01-11T04:26:26-07:00
The following classes have been added: DrawableTextInterlineSpacing, DrawableTextInterwordSpacing, DrawableTextKerning. And the following methods have been added to the Image class: textInterlineSpacing, textInterwordSpacing, textKerning. These changes will be available in ImageMagick 6.8.8-2.
````

source : https://www.imagemagick.org/discourse-server/viewtopic.php?t=24716#p106755

**Comment by stonebig** [comment]
well, I discover now that scikit-image was on the process of shifting to FreeImage inside imageio, so my demand seems not pertinent. sorry for the noise
